card/mod/Modfile

```diff
@@ -21,12 +18,15 @@ mod 'media'
 mod 'search'
 mod 'standard'
 mod 'rules'
-mod 'edit'
-mod 'date'
 mod 'account'
 mod 'navbar'
 mod 'bootstrap'
 mod 'google_analytics'
-mod 'recaptcha'
+gem_mod 'recaptcha'
 gem_mod 'markdown'
+gem_mod 'date'
+gem_mod 'ace_editor'
+gem_mod 'prosemirror_editor'
+gem_mod 'tinymce_editor'
+gem_mod 'edit'
```

Comment:
Based on what we learned the other day about how gem_mods are loaded, these may not be necessary?

Reply:
Yeah, not at the end anyway. I just moved them back to the original order.